### PR TITLE
Add types to some files in Pyodide

### PR DESCRIPTION
**Pull request by garrettgu10**

### CODE REVIEW

File: src/pyodide/internal/builtin_wrappers.ts [hunk 1]
```diff
@@ -84,7 +83,7 @@ function checkCallee() {
  * `convertJsFunctionToWasm` or `loadModule` in `pyodide.asm.js`, `false` if not. This will set
  * the `stack` field in the error so we can read back the result there.
  */
-function prepareStackTrace(_error, stack) {
+function prepareStackTrace(_error: Error, stack: Array<StackItem>) {
```
Comment (hoodmane) [inline src/pyodide/internal/builtin_wrappers.ts:86]: Maybe:
```suggestion
function prepareStackTrace(_error: Error, stack: StackItem[]) {
```

File: src/pyodide/internal/builtin_wrappers.ts [hunk 2]
```diff
@@ -106,7 +105,7 @@ function prepareStackTrace(_error, stack) {
   }
 }
 
-export async function wasmInstantiate(module, imports) {
+export async function wasmInstantiate(module: any, imports: WebAssembly.Imports) {
```
Comment (hoodmane) [inline src/pyodide/internal/builtin_wrappers.ts:108]:
```suggestion
export async function wasmInstantiate(module: WebAssembly.Module | Uint8Array, imports: WebAssembly.Imports) {
```

**Review by hoodmane — APPROVED** [review]
Thanks @garrettgu10

**Review by dom96 — APPROVED** [review]
awesome to see types here!

**Pull request merged**